### ISSUE TITLE
Debugger problems with unsaved reformat and breakpoint in block

### ISSUE THREAD

**Issue by LisaAlmarode** [ericwinger/Jade#609]
If you have a breakpoint within a block, and you are in the debugger, and you reformat the code, but don't save - click off to another method on the stack (answering no to the question), and then you return to the original method....

The reformatting was sticky (it did not revert to the original source code)

The highlight for the debugger may be in the wrong place

If you are unlucky, it says illegal offset and then gives you this:

![image](https://user-images.githubusercontent.com/13142166/67704143-d152dc80-f971-11e9-918f-d3eb6787deff.png)

====

The method source that I used was
```
test_highlighBreaks4

self 
should: 
[ String new 
at: 
4 
] 
raise: 
[ 
Error 
]
```

And the break point was on step point 8, line 10.  The breakpoint wasn't hit, the at: 4 is an error.  Debugged the test, and opened the debugger.  Go to the context with this test method.

reformat source.  Don't save.  Click on a different method.  Answer no to stay on unsaved method. click back on test method.  Bad things commence to occur.

**Comment by ericwinger** [comment]
Have a safe fix in 3.0.86 ready to go.

**Comment by LisaAlmarode** [comment]
Verified in 3.0.86-release candidate
